tigergraph/queries/bi-9.gsql

```diff
@@ -1,33 +1,28 @@
-CREATE OR REPLACE DISTRIBUTED QUERY bi9(DATETIME startDate, DATETIME endDate) SYNTAX v2 {
+CREATE OR REPLACE DISTRIBUTED QUERY bi9(DATETIME startDate, DATETIME endDate) {
   TYPEDEF TUPLE <UINT personId, STRING personFirstName, STRING personLastName, UINT threadCount, UINT messageCount> RESULT;
   HeapAccum<RESULT>(100, messageCount DESC, personId ASC) @@result;
   SetAccum<VERTEX<Person>> @root;
   SumAccum<UINT> @messageCount;
   SumAccum<UINT> @threadCount;
+  MaxAccum<UINT> @postId;
 
-  INT startEpoch = datetime_to_epoch(startDate) * 1000;
-  INT endEpoch = datetime_to_epoch(endDate) * 1000;
-  threads = SELECT p FROM Post:p WHERE p.creationDate BETWEEN startEpoch AND endEpoch;
-  persons =
-    SELECT p
-    FROM threads:t -(HAS_CREATOR>)- Person:p
+  threads = SELECT p FROM Post:p WHERE p.creationDate > startDate AND p.creationDate < endDate;
```

Comment:
The schema here is different, creationDate is an INT and includes milliseconds. You need to modify it.